class.jetpack.php

```diff
@@ -6929,25 +6930,27 @@ public static function build_raw_urls( $url ) {
 	 * Stores and prints out domains to prefetch for page speed optimization.
 	 *
 	 * @param mixed $new_urls
+	 * @param string | $relation_type | resource hint type
 	 */
-	public static function dns_prefetch( $new_urls = null ) {
+	public static function dns_prefetch( $new_urls = null, $relation_type ) {
```

Comment:
Could you add a default value here for `$relation_type`, to avoid issues when the function is called without a second parameter?

Reply:
Yep, just caught that - commited up with this change, setting a default value of `$relation_type = 'dns-prefetch'`

Reply:
The default value should be all you need; since you have a default `ab98c7244215e0306bee2acdfe57c9cf48f3b8ff` is not necessarily needed.

Reply:
Very true! Can get rid of that commit for now, as noted in another comment, it probably wouldn't make much sense to call `Jetpack::dns_prefetch` if you were wanting to `<link rel="preconnect">` as the function name would throw you off? Overtime you guys might want to look at changing from `Jetpack::dns_prefetch` to something like `Jetpack::relation_types` (probably could come up with a better name!!)

Reply:
@jeherve - I believe I have now reverted that commit: ab98c7244215e0306bee2acdfe57c9cf48f3b8ff